### PR TITLE
Documentation around the new Postman console.

### PR DESCRIPTION
**Pull request by arnitkun**
This pull request adds information around the new postman console, along with the screenshots and updates the relevant places in the docs.

### CODE REVIEW

File: v6/postman/launching_postman/navigating_postman.md [hunk 1]
```diff
@@ -53,6 +53,57 @@ Two consoles are available in Postman to see what's going on behind the scenes.
 
 * DevTools console - Provides diagnostic information during development. To learn how to access DevTools console logs, see [Debugging and logs](/docs/postman/sending_api_requests/debugging_and_logs/).
 
+## Postman Console
+
+Postman has its own console, which can be used to view all the HTTP requests and responses for the current session, along with any messages the user wants to log.
+
+    **Note:** The new console comes with Postman app version 7.10 and later. Older version will **NOT** receive the new console. Also, once the app is updated to 7.10, is is **NOT** possible to revert.
```
Comment (kaustavdm) [inline v6/postman/launching_postman/navigating_postman.md]: Line shouldn't have leading whitespace.

File: v6/postman/launching_postman/navigating_postman.md [hunk 2]
```diff
@@ -53,6 +53,57 @@ Two consoles are available in Postman to see what's going on behind the scenes.
...
+
+The header now contains 3 components:
+
+1. A search bar, making it possible to search for a specific request.
+2. Filter option, allowing requests to be filtered on the basis of their message type: log, info, warning, error.
+3. Clear button to clear the console screen.
+
+There is now support for 4 types of console messages:
+
+1. **Log:** console.log(“This is a log.”);
```
Comment (kaustavdm) [inline v6/postman/launching_postman/navigating_postman.md]: Codefence the `console.log()` blocks.

File: v6/postman/launching_postman/navigating_postman.md [hunk 3]
```diff
@@ -53,6 +53,57 @@ Two consoles are available in Postman to see what's going on behind the scenes.
...
+[![console labels](https://assets.postman.com/postman-docs/console_labels.png)](https://assets.postman.com/postman-docs/console_labels.png)
+
+The console now shows the following four types of information for each request:
+
+1. Network
+2. Request headers
+3. Response headers
+4. Response body
+
+[![console network inf](https://assets.postman.com/postman-docs/console_network_info.png)](https://assets.postman.com/postman-docs/console_network_info.png)
```
Comment (kaustavdm) [inline v6/postman/launching_postman/navigating_postman.md]: s/inf/info

File: v6/postman/launching_postman/navigating_postman.md [hunk 4]
```diff
@@ -53,6 +53,57 @@ Two consoles are available in Postman to see what's going on behind the scenes.
...
+
+**The Console footer:**
```
Comment (kaustavdm) [inline v6/postman/launching_postman/navigating_postman.md]: Shouldn't we use headings instead of bold?

Comment (arnitkun) [reply v6/postman/launching_postman/navigating_postman.md]: Indeed, it should be headings. Changed it.

File: v6/postman/launching_postman/navigating_postman.md (hunk 1 above)
Comment (arnitkun) [reply v6/postman/launching_postman/navigating_postman.md]: Older commit from #1749 , should have spotted it.

File: v6/postman/launching_postman/navigating_postman.md (hunk 3 above)
Comment (arnitkun) [reply v6/postman/launching_postman/navigating_postman.md]: corrected to "inf" to "info".

**Pull request merged**